iphone/CoreApi/CoreApi/Common/MWMGeoUtil.mm

```diff
@@ -31,10 +32,8 @@ @implementation Measure
 */
 
 - (NSString*) valueAsString {
-  if (self.value > 9.999)
-    return [NSString stringWithFormat:@"%.0f", self.value];
-  else
-    return [NSString stringWithFormat:@"%.1f", self.value];
+  auto outString = measurement_utils::ToStringPrecision(self.value, self.value >= 10.0 ? 0 : 1);
+  return [NSString stringWithCString:outString.c_str() encoding:[NSString defaultCStringEncoding]];
```

Comment:
stringWithUTF8String

Reply:
Noted! I will use `stringWithUTF8String` instead of `stringWithCString`.